rebasehelper/utils.py

```diff
@@ -658,6 +658,8 @@ def install_build_dependencies(spec_path=None, assume_yes=False):
         cmd = ['dnf', 'builddep', spec_path]
         if os.geteuid() != 0:
             cmd = ['pkexec'] + cmd
+            logger.warning("Authentication required to install build "
+                           "depenencies by command '%s'", ' '.join(cmd))
```

Comment:
I think it would make more sense to move this up by one line, so it prints the command without `pkexec` prepended.

Reply:
Also, there is a typo: `depenencies`. And this sound a little bit better to me: `Authentication required to install build dependencies using '%s'`.

Reply:
OK I will fix those. Thanks for the checking.
But actually what is the max line length for this project's rule?


There is a setting in pylintrc
```
# Maximum number of characters on a single line.
max-line-length=120
```

it is checked by c0301. But as we know, all the pylint convention check is disabled by `py.test --pylint-error-types=EF` right  now.
http://pylint-messages.wikidot.com/messages:c0301

Reply:
The limit is 120 characters. Yes, it's not enforced, but you shouldn't exceed it anyway.